### PR TITLE
Kaizen

### PR DESCRIPTION
**Pull request by maulerrr**

### CODE REVIEW

**Comment by maulerrr** [comment]
?

**Review by DiarCode — COMMENTED** [review]
Fix issues

File: backend/src/main/java/org/kaiteki/kaizen/api/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Chatbot API
+
+A simple Flask API for interacting with a conversational AI model. This API allows you to perform various natural language processing tasks, such as chat-based responses, text summarization, keyword extraction, and paraphrasing.
```
Comment (DiarCode) [inline backend/src/main/java/org/kaiteki/kaizen/api/README.md:3]: Rewrite readme for up to date information. its incorrect now, in response type and etc.

Better use swagger for documentation. In readme file only specify the guide on how to run project and important endpoints

**Review by DiarCode — APPROVED** [review]
Ok, now it is better. But there are still some improvements in next iterations in further development!

**Pull request merged**